XmacsLabs/mogan
# [24_16] Change <font> to <span> (focus on color)

## What
Replacing <font> tags with <span> tags with CSS styling.

## Why
The HTML5 standard no longer recommends the <font> tag. This tag was primarily used in earlier versions of HTML to define font styles directly within the HTML code. HTML5 recommends using CSS to control fonts and styles.

The relevant rules can be found in subsection 1.11.1 of https://html.spec.whatwg.org/#presentational-markup
![图片](https://github.com/user-attachments/assets/baa37076-da2e-4c45-ac33-bb98b93c622a)


## How to test your changes?
Convert 24_16.tm to html, and then check to see if the part of the html file about the font color has been changed. (The reason for not writing a test file is that the reviewer needs to see both the readability of the html with the naked eye and the changes in the html source, and opening the html directly on its own would be more intuitive than just looking at the results of a check test.)
